styles/main.css

```diff
@@ -54,7 +54,7 @@ footer {
     font-size: 12px;
     text-align: center;
     margin-bottom: 2em;
-    margin-top:auto;margin-top:auto;
+    margin-top:auto;

```

Comment:
This wasn't working because you hadn't set width-wrap to a flex container (see below)